bip-XXXX-Bech32_Encoded_Transaction_Position_References.mediawiki

```diff
@@ -0,0 +1,278 @@
+* The block height hasn't been minded yet.
+* The Transaction Position is more than the total transaction within the referenced block.
+Therefore, implementers must be careful not to display TxRef's to users prematurely:
+* Applications MUST NOT display TxRef's for transactions with less than 6 confirmations.
+* Application MUST show a warning for TxRef's for transactions with less than 100 confirmations.
+** This warning SHOULD state that in the case of a large reorganization, the TxRefs Displayed may point to a different transaction, or to no transaction at all.
+
+=== Human Display Formats ===
+This specification provides standardisation of encoding and namespacing for '''application display formats'''.
+
+If a display format doesn't comply with this standard, it MUST be called a '''non-standard application display format'''.
+
+==== Encoding ====
+* All display formats MUST be encoded with standard Bech32<ref>'''Why use Bech32 Encoding for Confirmed Transaction References?''' The error detection and correction properties of this encoding format make it very attractive. We expect that it will be reasonable for software to correct a maximum of two characters; however, we haven’t specified this yet.</ref> encoding as defined within the [https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki BIP 0173] specification.
+* The Human Readable Part of the Bech32 encoding MUST be "tx"<ref>'''Why use the same Human Readable Part for different systems or blockchains?''' The ‘tx1’ prefix is to namespace the Bech32 encoding to this specification. We use Magic Values internally to namespace between projects.</ref>.
+* All Non Bech32 alphabet characters after the Bech32 code separator MUST be ignored when phasing, (except for terminating characters).<ref>'''Why Strip all Non-Bech32 Alphabet Characters?''' We do not wish to expect the users to keep their TxRef's in good unicode form. We expect them to copy, paste, write by-hand, write in a mix of character sets, etc.  Parsers should automaticity correct for all of these sorts of errors. However, we only specify the removal of non-bech32-alphabet characters.
+</ref>
+
+==== Namespacing ====
+* Every non-compatible system MUST use a different '''magic code'''.
+* The magic code MUST be prefixed to the data payload, and included in the Bech32 encoding.
+* Applications MUST reject codes with unknown magic codes.
+* Applications MUST display an error for known, but unsupported magic codes.
+
+=== Magic Codes ===
+* Magic codes are either 5, 10, 15, or 20-bits long.
+
+* The length of the magic code is set by the first 5 bits:
+** 0x0 to 0x7: 5-bit long. (All Reserved<ref>'''Why reserve all single character Magic Codes?''' The authors of this specification see many potential Bitcoin Related uses for additional magic codes, such as with Sidechains and other Special User Facing Projects.</ref> for Bitcoin-Related Use, we have the special exception for Litecoin, that is also assigned a 5-bit code.)
```

Comment:
"Special User Facing Projects" => "special user facing projects".
This whole document has strange capitalization like that above, was there some intention behind that?